src/planwise/component/engine.clj

```diff
@@ -99,19 +101,97 @@
                           :demand-raster    demand-raster}]
     (debug "Source population demand:" source-demand)
     (let [processed-providers (compute-providers-demand providers props)
-          initial-demand      (demand/count-population demand-raster)
+          scenario-demand      (demand/count-population demand-raster)
           quartiles           (vec (demand/compute-population-quartiles demand-raster))
           update-providers    (compute-providers-demand providers (assoc props :update? true))]
       (raster/write-raster demand-raster (str "data/" raster-path ".tif"))
       (raster/write-raster (demand/build-renderable-population demand-raster quartiles) (str "data/" raster-path ".map.tif"))
       {:raster-path      raster-path
        :source-demand    source-demand
-       :pending-demand   initial-demand
-       :covered-demand   (- source-demand initial-demand)
+       :pending-demand   scenario-demand
+       :covered-demand   (- source-demand scenario-demand)
        :demand-quartiles quartiles
        :providers-data   (mapv (fn [[a b]] (merge a b)) (map vector processed-providers update-providers))})))
 
-(defn compute-scenario
+(defn sum-up
+  [coll k]
+  (reduce (fn [acum elem] (+ acum (k elem))) 0 coll))
```

Comment:
`mapsum` or `sum-map` would be more descriptive names. Also, `k` can be called `f` to denote that's it's a generic function.

Another way of writing this function would be: `(apply + (map k coll))`